crmsh/utils.py

```diff
@@ -2929,4 +2946,11 @@ def fatal(error_msg):
     handled by Context.run in ui_context.py
     """
     raise ValueError(error_msg)
+
+
+def is_two_nodes_cluster():
+    """
+    Check if current cluster has two nodes
+    """
+    return len(list_cluster_nodes()) == 2
```

Comment:
Please rename 'is_two_nodes_cluster()' to something like `is_2node_cluster_without_qdevice()`, and also add the qdevice check accordingly.

Reply:
So that means if there are 2 nodes cluster + qdevice running, it's the safe case which will not have the fencing race issue, and don't need `pcmk_delay_max`?

Reply:
You are right.